style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* the nav bar */
+nav {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding: 12px 30px;
+}
+
+/* logo */
+.logo h1 {
+  color: #6070ff;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+/* toggle-menu */
+.toggle-menu img {
+  width: 28px;
+}
+
+/* INTRODUCTION SECTION */
+.intro-section {
+  background: url(/assets/images/header-bg-mobile.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 80vh;
+  padding-left: 40px;
+  text-align: left;
+}
+
+/* HEADLINE */
+.headline {
+  font-size: 40px;
+  line-height: 54px;
+  font-weight: 800;
+  padding: 10px 0;
+  margin-top: 10px;
+  padding-top: 100px;
+}
```

Comment:
-[ ] Please make sure that you fix the margin on the right as the previous Code Reviewer asked to.
![image](https://user-images.githubusercontent.com/61919617/170337110-c897f6e6-9115-4c87-a0d8-e69519a84ea7.png)
![image](https://user-images.githubusercontent.com/61919617/170337176-4df1bb50-b1d6-4217-ae32-84a8c651da3c.png)